### ISSUE TITLE
overview page over existing credential-definitions

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#115]
## User story

1. As a {HR-employee}
2. I want / need {an overview page over existing credential definitions}
3. So that {I can have a look at them and later edit them or assign DIs to existing credential definitions}

## Acceptance criteria
* credential-definition-overview page (credDef-overview-page) exists in navigation bar and opens when user clicks on it
* table similar to schema-overview is displayed (adapted to the information that credential definitions possess)
  * name-column
  * status of the credential definition (active vs. archived)
* Edit button exists 
* "Add Digital Identity"-button exists for each credential definition displayed
  * will later link to a pop-up window #114 

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release
